pkgs/development/libraries/nvidia-optical-flow-sdk/default.nix

```diff
@@ -18,6 +18,11 @@ stdenv.mkDerivation {
     cp -R * $out/include
   '';
 
+  postFixup = ''
+    mkdir -p $out/nix-support
+    echo $pname >> "$out/nix-support/include-in-cudatoolkit-root"
+  '';
+
```

Comment:
This feels a tiny bit like magic -- is it possible to get the same effect by including the CUDA setup hook?

Reply:
Damn, this is exactly what `cudaPackages.markForCudatoolkitRootHook` does 🤦🏻 this already went to master though

Reply:
Would you mind submitting a fix for master and cherry-picking it here?

Reply:
Yes, I should